fido2/nfc.py

```diff
@@ -17,22 +17,23 @@ class CtapNfcDevice(CtapDevice):
     :cvar descriptor: Device descriptor.
     """
 
-    def __init__(self, descriptor, dev):
+    def __init__(self, descriptor, dev, no_card=False):
```

Comment:
Is `no_card` added only for the tests? I wouldn't change the API to ease testing, but provide a mocked `dev` with mocks for all the methods used in this constructor.

Reply:
ok, ill delete it